glean-core/src/upload/request.rs

```diff
@@ -37,6 +37,27 @@ impl PingRequest {
         }
     }
 
+    /// Get the uuid as an owned String.
+    pub fn uuid(&self) -> String {
+        self.uuid.to_owned()
+    }
+
+    /// Get the path as an owned String.
+    pub fn path(&self) -> String {
+        self.path.to_owned()
+    }
+
+    /// Get the body as a JSON formatted string.
+    pub fn body(&self) -> String {
+        serde_json::to_string(&self.body).unwrap()
+    }
+
+    /// Get the headers as a JSON formatted string.
+    pub fn headers(&self) -> String {
```

Comment:
Why does this need to be a json formatted string?

Reply:
Like I said in the other comment, I thought it made sense to serialize this as JSON. It will make it easier for platforms to deserialize it later.